src/main/java/org/jasig/portlet/widget/links/ResourceLinksConfigController.java

```diff
@@ -0,0 +1,68 @@
+package org.jasig.portlet.widget.links;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+import javax.portlet.PortletException;
+import javax.portlet.PortletMode;
+import javax.portlet.PortletRequest;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.portlet.bind.annotation.ActionMapping;
+import org.springframework.web.portlet.bind.annotation.RenderMapping;
+
+@Controller
+@RequestMapping("CONFIG")
+public class ResourceLinksConfigController extends ResourceLinksBaseController {
+
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
📓 Lombok provides a nice helper to normalize log usage
Ref: https://projectlombok.org/features/log